.github/workflows/ci.yml

```diff
@@ -72,6 +71,22 @@ jobs:
           cd ..
           echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$PWD/usr/lib" >> $GITHUB_ENV
           echo "TOOLCHAIN_OPTION=$TOOLCHAIN_OPTION -DCMAKE_PREFIX_PATH=$PWD/usr" >> $GITHUB_ENV
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Prepare OpenSSH Server Container for Testing
+        uses: docker/build-push-action@v2
+        with:
+          pull: true
+          load: true
+          push: ${{ github.event_name == 'push' }}
+          context: ./tests/openssh_server
+          tags: |
+            libssh2/openssh_server
```

Comment:
Just a heads up: it could be that this will cause the push to fail, because only the 2nd tag can be pushed.